cloudformation/database/serverless-database.yaml

```diff
@@ -7,6 +7,7 @@ Metadata:
       - Label:
           default: "Database Cluster Configuration"
         Parameters:
+          - ClusterName
```

Comment:
Was this necessary only for serverless v2? Or is this a general improvement to the template?

Reply:
General improvement - default, generated names could exceed the 63 character limit.